cmd/minishift/cmd/config/config.go

```diff
@@ -52,18 +52,21 @@ var settingsList []Setting
-	CPUs             = createConfigSetting("cpus", SetInt, []setFn{validations.IsPositive}, []setFn{RequiresRestartMsg}, true, nil)
-	Memory           = createConfigSetting("memory", SetString, []setFn{validations.IsValidMemorySize}, []setFn{RequiresRestartMsg}, true, nil)
-	DiskSize         = createConfigSetting("disk-size", SetString, []setFn{validations.IsValidDiskSize}, []setFn{RequiresRestartMsg}, true, nil)
-	VmDriver         = createConfigSetting("vm-driver", SetString, []setFn{validations.IsValidDriver}, []setFn{RequiresRestartMsg}, true, nil)
-	OpenshiftVersion = createConfigSetting("openshift-version", SetString, nil, nil, true, nil)
-	HostOnlyCIDR     = createConfigSetting("host-only-cidr", SetString, []setFn{validations.IsValidCIDR}, nil, true, nil)
-	DockerEnv        = createConfigSetting("docker-env", SetSlice, nil, nil, true, nil)
-	DockerEngineOpt  = createConfigSetting("docker-opt", SetSlice, nil, nil, true, nil)
-	InsecureRegistry = createConfigSetting("insecure-registry", SetSlice, nil, nil, true, nil)
-	RegistryMirror   = createConfigSetting("registry-mirror", SetSlice, nil, nil, true, nil)
-	AddonEnv         = createConfigSetting("addon-env", SetSlice, nil, nil, true, nil)
+	ISOUrl                = createConfigSetting("iso-url", SetString, []setFn{validations.IsValidISOUrl}, []setFn{RequiresRestartMsg}, true, nil)
+	CPUs                  = createConfigSetting("cpus", SetInt, []setFn{validations.IsPositive}, []setFn{RequiresRestartMsg}, true, nil)
+	Memory                = createConfigSetting("memory", SetString, []setFn{validations.IsValidMemorySize}, []setFn{RequiresRestartMsg}, true, nil)
+	DiskSize              = createConfigSetting("disk-size", SetString, []setFn{validations.IsValidDiskSize}, []setFn{RequiresRestartMsg}, true, nil)
+	VmDriver              = createConfigSetting("vm-driver", SetString, []setFn{validations.IsValidDriver}, []setFn{RequiresRestartMsg}, true, nil)
+	OpenshiftVersion      = createConfigSetting("openshift-version", SetString, nil, nil, true, nil)
+	HostOnlyCIDR          = createConfigSetting("host-only-cidr", SetString, []setFn{validations.IsValidCIDR}, nil, true, nil)
+	DockerEnv             = createConfigSetting("docker-env", SetSlice, nil, nil, true, nil)
+	DockerEngineOpt       = createConfigSetting("docker-opt", SetSlice, nil, nil, true, nil)
+	InsecureRegistry      = createConfigSetting("insecure-registry", SetSlice, nil, nil, true, nil)
+	RegistryMirror        = createConfigSetting("registry-mirror", SetSlice, nil, nil, true, nil)
+	AddonEnv              = createConfigSetting("addon-env", SetSlice, nil, nil, true, nil)
+	RemoteIPAddress       = createConfigSetting("remote-ip-address", SetString, nil, nil, true, nil)
+	RemoteSSHUser         = createConfigSetting("remote-ssh-username", SetString, nil, nil, true, nil)
+	SSHKeyToConnectRemote = createConfigSetting("ssh-key-to-connect-remote", SetString, nil, nil, true, nil)
```

Comment:
IMO it should be just `ssh-key`.  `--ssh-key-to-connect-remote` is very long to type. @gbraad WDYT?

Reply:
logically grouped first, but thi can be named `--[prefix]-sshkeyfile`